src/main/kotlin/dev/isxander/pronounify/utils/PronounManager.kt

```diff
@@ -65,22 +68,23 @@ object PronounManager {
             coroutineScope {
                 chunked.map {
                     async {
-                        try {
+                        val pronouns = try {
```

Comment:
This area is a bit hard to read, a comment or two wouldn't hurt, I think.
Notably, the actual value of the definition is a few lines lower down, it's not immediately clear what value `pronouns` will take. Explicitly saying it here at the top should help.